vmdb/app/models/policy_event.rb

```diff
@@ -1,7 +1,7 @@
 class PolicyEvent < ActiveRecord::Base
```

Comment:
Class PolicyEvent has a table that records all policy related events that have been raised.